### PR TITLE
Update README/CONTRIBUTING and npm package

### PR DESCRIPTION
**Pull request by bidoubiwa**
Before open PR: 
- FInish README
- Update Contributing

fixes: #24 -> release drafter
fixes: #23 -> SDK conform headers
fixes: #21 -> Update Contributing

### CODE REVIEW

File: .github/release-draft-template.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name-template: 'v$RESOLVED_VERSION 🌻'
```
Comment (curquiza) [inline .github/release-draft-template.yml:1]:
```suggestion
name-template: 'v$RESOLVED_VERSION 🚀'
```

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: Install dependencies
+        run: yarn install
+      - name: Build
+        run: yarn build
+      - name: Check release validity
+        run: sh scripts/check-release.sh
```
Comment (curquiza) [inline .github/workflows/publish.yml:21]:
```suggestion
        run: sh ./github/scripts/check-release.sh
```

File: README.md [hunk 3]
```diff
@@ -1,61 +1,257 @@
...
+  <a href="https://github.com/meilisearch/MeiliSearch">MeiliSearch</a> |
+  <a href="https://docs.meilisearch.com">Documentation</a> |
+  <a href="https://slack.meilisearch.com">Slack</a> |
+  <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
+  <a href="https://www.meilisearch.com">Website</a> |
+  <a href="https://docs.meilisearch.com/faq">FAQ</a>
+</h4>
 
+<p align="center">
+  <a href="https://www.npmjs.com/package/meilisearch"><img src="https://img.shields.io/npm/v/meilisearch.svg" alt="npm version"></a>
```
Comment (curquiza) [inline README.md:17]:
```suggestion
  <a href="https://www.npmjs.com/package/strapi-plugin-meilisearch"><img src="https://img.shields.io/npm/v/strapi-plugin-meilisearch.svg" alt="npm version"></a>
```

File: README.md [hunk 4]
```diff
@@ -1,61 +1,257 @@
...
+  <a href="https://github.com/meilisearch/meilisearch-js/actions"><img src="https://github.com/meilisearch/meilisearch-js/workflows/Tests/badge.svg" alt="Tests"></a>
```
Comment (curquiza) [inline README.md:18]:
```suggestion
  <a href="https://github.com/meilisearch/strapi-plugin-meilisearch/actions"><img src="https://github.com/meilisearch/strapi-plugin-meilisearch/workflows/Tests/badge.svg" alt="Tests"></a>
```

File: README.md [hunk 5]
```diff
@@ -1,61 +1,257 @@
...
   <a href="https://github.com/prettier/prettier"><img src="https://img.shields.io/badge/styled_with-prettier-ff69b4.svg" alt="Prettier"></a>
   <a href="https://github.com/meilisearch/meilisearch-js/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
```
Comment (curquiza) [inline README.md:20]:
```suggestion
  <a href="https://github.com/meilisearch/strapi-plugin-meilisearch/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
```

File: README.md [hunk 6]
```diff
@@ -1,61 +1,257 @@
...
   <a href="https://app.bors.tech/repositories/28762"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
```
Comment (curquiza) [inline README.md:21]:
```suggestion
  <a href="https://app.bors.tech/repositories/32218"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
```

File: scripts/check-release.sh [hunk 7]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Checking if current tag matches the package version
+current_tag=$(echo $GITHUB_REF | tr -d 'refs/tags/v')
+file_tag=$(grep '"version":' package.json | cut -d ':' -f 2- | tr -d ' ' | tr -d '"' | tr -d ',')
+if [ "$current_tag" != "$file_tag" ]; then
+  echo "Error: the current tag does not match the version in package file(s)."
```
Comment (curquiza) [inline scripts/check-release.sh:7]:
```suggestion
  echo "Error: the current tag does not match the version in the package.json."
```

**Review by curquiza — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1,61 +1,257 @@
...
+<img src="./assets/no_reload_needed.png" alt="Indexed collections are hooked" width="600"/>
+</p>
+
+### 🕵️‍♀️ Start Searching
+
+Once you have a collection containing documents indexed in MeiliSearch, you can [start searching](https://docs.meilisearch.com/learn/getting_started/quick_start.html#search).
+
+Using the above credentials, the following code shows how to search on one of your collections:
+
+To search in MeiliSearch, you can use the [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) SDK that integrates a whole search interface, or our library [meilisearch-js](https://github.com/meilisearch/meilisearch-js).
```
Comment (curquiza) [inline README.md:137]:
```suggestion
To search in MeiliSearch, you can use the [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) libreary that integrates a whole search interface, or our [meilisearch-js](https://github.com/meilisearch/meilisearch-js) SDK.
```

File: README.md [hunk 9]
```diff
@@ -1,61 +1,257 @@
...
+
+#### ⚡️ Using Instant meiliSearch
+
+You can have a front up and running in record time with [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch).
+
+<p align="center">
+<img src="./assets/obrigado.gif" alt="Restaurant demo" width="600"/>
+</p>
+
+In Instant MeiliSearch, you only have to provide your credentials and index name (`restaurant` is the index name in our example).
```
Comment (curquiza) [inline README.md:147]:
```suggestion
In Instant MeiliSearch, you only have to provide your credentials and index name (_uid_). `restaurant` is the index name in our example.
```

File: README.md [hunk 10]
```diff
@@ -1,61 +1,257 @@
-# MeiliSearch in Strapi Plugin 🔎
-
-Index your Strapi collections into a MeiliSearch instance. The plugin listens to modifications made on your collections and update MeiliSearch accordingly.
+<h1 align="center">MeiliSearch Strapi Plugin</h1>
 
 <p align="center">
+  <img src="https://res.cloudinary.com/meilisearch/image/upload/v1587402338/SDKs/meilisearch_js.svg" alt="MeiliSearch-JavaScript" width="200" height="200" />
```
Comment (curquiza) [inline README.md:4]:
```suggestion
  <img src="https://raw.githubusercontent.com/meilisearch/integration-guides/main/assets/logos/meilisearch_strapi.svg" alt="MeiliSearch-Strapi" width="200" height="200" />
```

File: README.md [hunk 11]
```diff
@@ -1,61 +1,257 @@
...
 
-Install Strapi with this **Quickstart** command to create a Strapi project instantly:
+This command will install the required dependencies and launch the app in development mode. You should be able to reach it on the [port 8000 of your localhost](http://localhost:8000/admin/).
 
+### Create Strapi project
 
-#### Create strapi project
+Install Strapi with this [quickstart]((https://strapi.io/documentation/developer-docs/latest/getting-started/quick-start.html)) command to create a Strapi project instantly:
 
 - Use **yarn** to install the Strapi project (recommended). [Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/).)
```
Comment (curquiza) [inline README.md:256]:
```suggestion
- Use **yarn** to install the Strapi project (recommended). [Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/).
```

**Review by curquiza — APPROVED** [review]

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**